go/store/prolly/map.go

```diff
@@ -71,6 +72,28 @@ func NewMapFromTuples(ctx context.Context, ns NodeStore, keyDesc, valDesc val.Tu
 	return m, nil
 }
 
+func DiffMaps(ctx context.Context, from, to Map, cb DiffFn) error {
+	differ, err := treeDifferFromMaps(ctx, from, to)
+	if err != nil {
+		return err
+	}
+
+	for {
+		var diff Diff
+		if diff, err = differ.Next(ctx); err != nil {
+			break
+		}
+
+		if err = cb(ctx, diff); err != nil {
```

Comment:
Nit(?): Is it weird to let this return `io.EOF` and not forward it along to the caller?